### ISSUE TITLE
IOS XCode, Build failed, MoPub.h file not found

### ISSUE THREAD

**Issue by agrobost** [mopub/mopub-unity-sdk#161]
I have a problem with MoPub to build on xocde.

- [ X] I am submitting a bug report for existing functionality
- [X ] I visited https://developers.mopub.com/ and found no answer
- [X ] I checked https://twittercommunity.com/c/advertiser-api/mopub to make sure that this issue has not already been filed
- [X ] I checked to make sure that this issue has not already been filed

#### Steps to reproduce the behavior:
Build and run on IOS

#### Expected behavior:
Build unity success & build Xcode success

#### Observed behavior:
Pod failed, but when I change swift version  from 5.0 5 to 5.0 in Xcode, pod install works
After I double click on Unity-iPhone.xcworkspace, and when I build I get this error: MoPub.h file not found

#### Evidence:  
![2021-01-05 00_24_53-macOS Big Sur - VMware Workstation 16 Player (Non-commercial use only)](https://user-images.githubusercontent.com/14184466/103589798-87a64000-4eec-11eb-9316-4056188d4f7f.png)  
![2021-01-05 00_23_39-Clipboard](https://user-images.githubusercontent.com/14184466/103590069-35b1ea00-4eed-11eb-9622-3425d73e6bdd.png)

**Comment by agrobost** [comment]
EDIT:
The pod install error is due to the script 
MoPubPostBuildiOS.cs (line 45)
```csharp
project.AddBuildProperty(target, "SWIFT_VERSION", "5");
```
When I replace by 
```csharp
project.AddBuildProperty(target, "SWIFT_VERSION", "5.0");
```
Everything works well !

**Comment by p158276** [comment]
Hi @agrobost , 
Thanks for reaching out. This issue may be the Duplicate of #151. Can you try the workaround there and see if that work for you?

Yoyo Lin,
MoPub Team

**Comment by agrobost** [comment]
Hi @p158276 , thank you very much ! The workaround fix my issue, but I get a new error (from MoPub):
```
Cannot use '@try' with Objective-C exceptions disabled
```
I fixed this problem by changing "Enable Object-C Exceptions" from false to yes in Xcode settings

After two days I finally managed to build on IOS! 😃

**Comment by agrobost** [comment]
For those who encounter difficulties here is the complete workaround for build mopub on ios:

1/ In MoPubPostBuildiOS.cs (line 45) replace :
```csharp
project.AddBuildProperty(target, "SWIFT_VERSION", "5");
```
by
```csharp
project.AddBuildProperty(target, "SWIFT_VERSION", "5.0");
```

2/ Follow this workaround: https://github.com/mopub/mopub-unity-sdk/issues/151#issue-747782146

3/ In Xcode, change "Enable Object-C Exceptions" from **false** to **yes** in Xcode settings

**Comment by kogodm** [comment]
tks !

**Comment by CodeMasterYi** [comment]
@MoPub you need him!🐂🍺

**Comment by cathei** [comment]
I think you should not close the issue if it's not actually fixed in releases?

**Comment by CodeMasterYi** [comment]
> I think you should not close the issue if it's not actually fixed in releases?

right.

**Comment by p158276** [comment]
Hi @cathei @CodeMasterYi , 
Thanks for bringing it up! We are currently confirming whether this actually solved the problem among all Unity version developing environment. If everything goes well, we'll implement these changes in the coming SDK. We'll open this issue again and follow up here if there's any update.

Yoyo Lin 
MoPub Team

**Comment by p158276** [comment]
Fixed in v[5.16.0](https://github.com/mopub/mopub-unity-sdk/commit/ee33ae47a05bc2677262cf87673dd3a6fed37d81#diff-e8b711a273c4b593e3c5b248afb5c53068b38faf006d861d8ab6f6268105b6bcR49). Let us know if you continue seeing the issue!
